conferences/abstracts/RSECon19-BinderHub-workshop-application.md

```diff
@@ -0,0 +1,151 @@
+
+"Build a BinderHub for hosting Reproducible Software in the Cloud"
+
+### Authors
+
+* [Sarah Gibson](https://www.turing.ac.uk/people/researchers/sarah-gibson), ([The Alan Turing Institute](https://www.turing.ac.uk/)) on behalf of **[The Turing Way](https://github.com/alan-turing-institute/the-turing-way) Collaboration**.
+
+### Abstract
+
+_(1250 characters max, current count: 1244 [please update if you edit])_
+
+BinderHub is a Cloud-based, multi-user server technology that captures the computing environment of a GitHub repository within a Docker image.
+It then provides an interactive browser from which code can be run in the environment under which it was developed without placing installation responsibilities on the user.
+A unique URL to this browser is generated making the software easily sharable with collaborators.
+BinderHubs can be built anywhere in the world and are Cloud-neutral.
+Building a locally hosted BinderHub is advantageous as it would facilitate the use of private repositories and make available a greater variety of computing resources - facilities which are not currently available on the public Binder service.
+
+By harnessing the power of Cloud computing and building a BinderHub with expanded capabilities, scientific analyses can become more accessible and transparent to the wider community.
+BinderHubs can showcase the reproducibility of software as well as being a valuable teaching or demonstration tool.
+
+In this workshop, attendees will learn how to deploy a BinderHub on Microsoft Azure and connect with others who have or are interested in doing so.
+Attendees should be familiar with Azure Cloud computing and the command line.
+
+### Audience
+
+_Would your target audience be required to have any prerequisite skills/background knowledge e.g. knowledge of a particular language?_
+
+This workshop is most suitable for participants who are interested in maintaining a service to support researchers, for example at their institute.
+They do not need to have access to organisational Cloud computing resources to join the workshop.
+Attendees must be comfortable working on the command line.
+A familiarity with Microsoft Azure will make participating in the workshop a little easier, but it is not a prerequisite.
+
+### Outcomes
+
+_How will your attendees benefit from your session? What do you expect them to gain/learn?_
+
+_(600 characters max, current count: 524 [please update if you edit])_
+
+Attendees to this workshop will learn how to deploy a BinderHub on Microsoft Azure Cloud platform.
+They will learn the multiple applications of a BinderHub as a teaching or demonstration tool; the technology powering a BinderHub; and why reproducibility is important in research.
+Attendees will be provided with enough knowledge to make an informed decision on whether building their own BinderHub is advantageous to their goals and connect with other RSEs/researchers who have built a BinderHub or are interested in doing so.
```

Comment:
```suggestion
Attendees will be provided with enough knowledge to make an informed decision on whether operating their own BinderHub is advantageous to their goals and connect with other RSEs/researchers who have built a BinderHub or are interested in doing so.
```
What do you think?

Reply:
Nice. Shows that it's not just the case of building it!